### PR TITLE
Peggy Wang's Business Site Submission

### PR DESCRIPTION
**Pull request by pwangy**

### CODE REVIEW

**Review by HIPPIEKICK — APPROVED** [review]
Nice work with the hero video, and the +/- icons changing according to the active state. It's good to always strive for responsiveness so when you have time, take a look at your page in mobile mode. Well done Peggy!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    </div>
+
+    <!-- Hero -->
+    <header>
+      <video autoplay loop muted preload playsinline>
+        <source src="./video_hd.mp4" type="video/mp4" />
+        <p>
+          This browser doesn't support video. Here's a
+          <a href="./video_hd.mp4">link to the video</a> instead.
+        </p>
```
[the remark this answers is not in the document]
Comment (pwangy) [reply index.html:32]: updated CSS to specify a background color here.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  margin-left: 16px;
+  height: 5px;
+  width: 30px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+h2 {
+  font-size: 25px;
+  font-weight: none;
```
[the remark this answers is not in the document]
Comment (pwangy) [reply style.css:40]: fixed!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+nav a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+a:hover {
+  background-color: #fff;
+  color: #141452;
+  padding: 5px;
```
[the remark this answers is not in the document]
Comment (pwangy) [reply style.css:59]: padding removed

File: index.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+        class="form"
+        action="http://httpbin.org/anything"
+        method="POST"
+        target="_blank"
+      >
+        <h3>Reserve your spot</h3>
+        <div class="form-group">
+          <label for="nameValue">Name</label>
+          <input
+            id="nameValue"
```
[the remark this answers is not in the document]
Comment (pwangy) [reply index.html:55]: keeping ids in, i think that's what shows up in the httpbin.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+        <p class="headline">Let’s cook something up!</p>
+        <p class="header-blurb">
+          Because: Food is a community cooking school. We believe that life is
+          too short for boring food. Join us and learn something new.
+        </p>
+      </div>
+    </header>
+
+    <!-- Signup form -->
+    <section class="form-accordian">
```
[the remark this answers is not in the document]
Comment (pwangy) [reply index.html]: updated this class name.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+  overflow: hidden;
+  object-fit: cover;
+  object-position: center center;
+}
+
+/* Container for accordian & form */
+.form-accordian {
+  display: flex;
+  flex-direction: row-reverse;
+}
```
[the remark this answers is not in the document]
Comment (pwangy) [reply style.css]: Whew! It took some time to rework a bunch of this code to be mobile-first. Done!

**Pull request closed without merging**